src/LanguageSelect.tsx

```diff
@@ -1,26 +1,22 @@
 import { Select } from "baseui/select";
 import { useState } from "react";
 import { useTranslation } from "react-i18next";
-import languages from "./translations/resources";
+import languages, { Language } from "./translations/resources";
 import { enqueueAnalyticsEvent } from "./utils/analytics";
 
+
 const LanguageSelect = () => {
-  const langCode = localStorage.getItem("i18nextLng") || "en";
```

Comment:
Can you also add back in the localStorage logic since we want to persist user selection as per #70. With your current code, the dropdown will always have English selected on a page refresh.

Reply:
Ah very weird that it shows up as a delete. I think I just need to re-merge with master.